server/package.json

```diff
@@ -1,38 +1,38 @@
 {
-	"name": "@rescript/language-server",
-	"description": "LSP server for ReScript",
-	"version": "1.56.0",
-	"author": "ReScript Team",
-	"license": "MIT",
-	"bin": {
-		"rescript-language-server": "./out/cli.js"
-	},
-	"keywords": [
-		"ReScript",
-		"LSP",
-		"Language Server"
-	],
-	"files": [
-		"out/*",
-		"analysis_binaries",
-		"README.md"
-	],
-	"engines": {
-		"node": "*"
-	},
-	"homepage": "https://github.com/rescript-lang/rescript-vscode/server/README.md",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/rescript-lang/rescript-vscode",
-		"directory": "server"
-	},
-	"bugs": {
-		"url": "https://github.com/rescript-lang/rescript-vscode/issues"
-	},
-	"dependencies": {
-		"chokidar": "^3.5.1",
-		"vscode-jsonrpc": "^8.0.1",
-		"vscode-languageserver": "^8.0.1",
-		"vscode-languageserver-protocol": "^3.17.1"
-	}
+  "name": "@rescript/language-server",
```

Comment:
for some reason this was tab formatted (and in the server it's spaces - I think spaces is more standard)